README.md

```diff
@@ -6,6 +6,45 @@ CNCF Storage Working Group
 
 Explore cloud native storage technology and concepts.
 
+## Why is storage important to cloud native
+
+- There’s no such thing as a stateless architecture, applications store state somewhere
+- Storage further enables specific cloud native patterns
+- Interoperating with storage increases cloud native’s relevance and leads to better applications
+
+## Mission statement
+
+In support of cloud native computing, the SWG intends to enable a thriving storage eco-system that is vendor and platform neutral and interoperable for applications
+
+## Responsibilities
+
+- Make sense of storage eco-system supporting cloud native environments
+- Review storage projects
+- Assist storage projects in TOC project nomination process
+
+## Governance
+
+The storage working group is an open group of community members with a leading representative as defined by the CNCF TOC (Ben Hindman). It meets regularly and members provide input and opinions about storage topics. Leadership makes decisions on SWG perspectives to be delivered to the TOC. 
+
+## What is Cloud Native Storage
+Cloud Native Storage is a classification for storage resources that are interoperable for applications in cloud native environments. It is not necessarily a cloud native system itself, but supports decoupling of storage services from applications. Interoperability is possible through standard storage interfaces. These resources are reliable, consumed on-demand, and self-provisioned. Resources can be elastic in both capacity and performance utilization is measured to enable optimal and efficient use. The consumption experience for applications is seamless and transparent. 
```

Comment:
I do not think we should be coining a term like "cloud-native storage".

The perceived need for this definition here implies that the definition of "cloud native" in the CNCF charter (https://www.cncf.io/about/charter/) is insufficient. We should refine that to reflect our actual goals and values, rather than define what cloud native means for each category of infrastructure, application, or tool. Many of the points here would be useful as part of that refinement, and aren't specific to storage.

Additionally, we have to recognize that Cloud Native is aspirational. We are years away from achieving full automation of all infrastructure, higher-level services, and operations.